pods/configuration/connect-to-ide.mdx

```diff
@@ -0,0 +1,205 @@
+1. Generate an SSH key using this command on your local terminal:
+
+   ```sh
+   ssh-keygen -t ed25519 -C "YOUR_EMAIL@DOMAIN.COM"
+   ```
+
+2. To retrieve your public SSH key, run this command:
+
+    ```sh
+    cat ~/.ssh/id_ed25519.pub
+    ```
+    This will output something similar to this:
+
+    ```sh
+    ssh-ed25519 AAAAC4NzaC1lZDI1JTE5AAAAIGP+L8hnjIcBqUb8NRrDiC32FuJBvRA0m8jLShzgq6BQ YOUR_EMAIL@DOMAIN.COM
+    ```
+
+3.  Copy and paste the output into the **SSH Public Keys** field in your [RunPod user account settings](https://www.runpod.io/console/user/settings).
+
+<Warning>
+To enable SSH access, your public key must be present in the `~/.ssh/authorized_keys` file on your Pod. If you upload your public key to the settings page before your Pod starts, the system will automatically inject it into that file at startup.
+
+If your Pod is already running when you upload the key, the system will not perform this injection. To enable SSH access, you'll need to either restart your Pod, or manually add your public key to the `~/.ssh/authorized_keys` file inside your Pod.
+</Warning>
+
+## Step 3: Deploy a Pod
+
+Next, deploy the Pod you want to connect to. For detailed deployment instructions, see [Manage Pods -> Create a Pod](/pods/manage-pods#create-a-pod).
+
+To connect with VSCode/Cursor, your Pod template must support SSH over exposed TCP. To determine whether your Pod template supports this, during deployment, after selecting a template, look for a checkbox under **Instance Pricing** labeled **SSH Terminal Access** and make sure it's checked.
+
+<Tip>
+All official Runpod Pytorch templates support SSH over exposed TCP.
+</Tip>
+
+
+## Step 4: Configure SSH for your IDE
+
+Next, you'll configure SSH access to your Pod using the Remote-SSH extension. The instructions are different for VSCode and Cursor:
+
+<Tabs>
+
+  <Tab title="VSCode">
+   1. From the [Pods](https://www.runpod.io/console/pods) page, select the Pod you deployed.
+   2. Select **Connect**, then select the **SSH** tab.
+   3. Copy the second command, under **SSH over exposed TCP**. It will look similar to this:
+   
+      ```bash
+      ssh root@123.456.789.80 -p 12345 -i ~/.ssh/id_ed25519
+      ```
+
+      <Warning>
+      If you only see one command under SSH, then SSH over exposed TCP is not supported by your selected Pod template. This means you won't be able to connect to your Pod directly through VSCode/Cursor, but you can still connect using [basic SSH](/pods/connect-to-a-pod#basic-ssh-connection) via the terminal.
+      </Warning>
+
+   4. In VSCode, open the **Command Palette** (Ctrl+Shift+P or Cmd+Shift+P) and choose **Remote-SSH: Connect to Host**, then select **Add New SSH Host**.
+   5. Enter the copied SSH command from step 3 and press **Enter**. This will add a new entry to your SSH config file. It will look similar to this:
```

Comment:
Let's add the example command here?
```
ssh root@213.173.108.101 -p 31400 -i ~/.ssh/id_ed25519
```

Just so that someone has an idea as to what they are copying